account/domain/account.go

```diff
@@ -25,22 +36,39 @@ func (id AccountID) String() string {
 	return string(id)
 }
 
+func validateEmail(email string) error {
+	_, err := mail.ParseAddress(email)
+	return err
+}
+
 // Account is a type that represent a user account
 // ID could be later change by a UUID
 type Account struct {
 	ID        AccountID `json:"id"`
 	Firstname string    `json:"firstname"`
 	Lastname  string    `json:"lastname"`
-	Email     string    `json:"email"`
-	Admin     bool      `json:"admin,omitempty"`
-	Address   Address   `json:"address,omitempty"`
+	// Review: Replace by net/email.Address ?
```

Comment:
I don't think using the `net/mail` Address type is a good idea since it is just a name string & an email string. We could just extract the Address.Address's string.

Reply:
I agree